etc/kayobe/environments/ci-multinode/kolla/globals.yml

```diff
@@ -42,6 +42,8 @@ nova_backend_ceph: "yes"
 neutron_bridge_name: "{{ vxlan_interfaces[0].device }}-ovs"
 neutron_external_interface: "{{ vxlan_interfaces[0].device }}"
 
+neutron_dns_domain: "{{ root_domain }}."
```

Comment:
Something for the upgrade notes?

Reply:
reno or docs?

Reply:
It looks like this new variable is required to keep an existing feature working? If so, upgrade docs.

Reply:
Added some docs